Gemfile.lock

```diff
@@ -1,44 +0,0 @@
-GEM
```

Comment:
Oh, was the `Gemfile.lock` the problem? I was thinking last night after seeing you working on this that maybe since we are using `jruby-head` perhaps bundler doesn't know how to resolve it. But removing it entirely seems like a good solution...this repo isn't a gem so it's not really necessary. 👍